package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "dotcms-ui",
-    "version": "22.3.0-next.16",
+    "version": "22.3.0-rc.4",
```

Comment:
is these version correct ?

Reply:
no, @dsilvam this was changed on march 14

Reply:
The version in master should be kept